### PR TITLE
Hand_Gesture_Recognition

### PR DESCRIPTION
**Pull request by mihiragarwal92**
This is a python project based on various modules.

### CODE REVIEW

File: Python/Hand_gesture_recognition/readme
Comment (aravindvnair99) [inline Python/Hand_gesture_recognition/readme:58]: @mihiragarwal92 you sure you want to put your email publicly?

**Comment by mihiragarwal92** [comment]
Why didn't you approved?

**Comment by aravindvnair99** [comment]
> Why didn't you approved?

@mihiragarwal92 Please check the thread. Waiting for your confirmation, post which there will be a review of your code and only then will it be approved if everything looks good. This is currently pending on you.

**Comment by mihiragarwal92** [comment]
Sir,
I've no issues with email being public. However, if it feels inappropriate
to you I'll surely remove it.
Kindly revert back with your feedback.

**Comment by jahnavib26** [comment]
Hello Aravind and Mihir!

Apologies for the late reply. I'll review and confirm the status by the end
of the day.
Thank you!

Regards,
Jahnavi B

**Comment by mihiragarwal92** [comment]
Kindly approve the pr for the hacktoberfest  as soon as possible .Previously marked as stale.

**Review by jahnavib26 — APPROVED** [review]

**Review by jahnavib26 — APPROVED** [review]
Everything looks good! I will be approving it.

**Review by jahnavib26 — APPROVED** [review]

**Review by jahnavib26 — APPROVED** [review]

**Pull request merged**
